packages/gatsby-source-mongodb/package.json.lerna_backup

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
I'll assume these lerna_backup files got added by accident. I'll delete them and then merge